src/Components/Product/Filter.js

```diff
@@ -0,0 +1,105 @@
+import { useData } from "../../Context/DataContext/DataContext"
+
+export const Filter = () =>{
+    const{state : {categories, sort, price}, dispatch, category} = useData()
+    return(
+        <div>
+            <div class=""> 
+                <div class="flex pd-1 gap-4">
+                    <span>Filter</span>
+                    <span class="filter-text"
+                        onClick={()=>{
+                            dispatch({
+                                type:'CLEAR_ALL',
+                                payload: 'clear'
+                            })
+                        }}>
+                        Clear All
+                    </span>
+                </div>
+            <hr />
+            <div class="pd-1">
+                <p class="text-dark">Price</p>
+                    <input 
+                        type="range"
+                        min="1000"
+                        max="15000"
+                        step="20"
+                        value={price}
+                        label={price}
```

Comment:
`input` elements do have a `label` attribute?

Reply:
I forgot to remove it thanks for pointing it out.